paima-demo/godot-cip-30-frontend/cip_30_callbacks.gd

```diff
@@ -0,0 +1,68 @@
+extends Node
+# If `RefCounted` is used, then GDScript callbacks stop working - they are not called at all.
+# Probably coz references to callbacks are lost (see below).
+# (?) Alternative is to keep reference to `RefCounted` on root node (main.gd)
```

Comment:
Not exactly sure how `Node` here fixes the issue. Maybe there is some more lightweight class that can handle this as well.